worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
 		size_t GetSenderReportCount();
 		size_t GetReceiverReportCount();
 		void Dump();
 		void AddSenderReport(SenderReport* report);
 		void AddReceiverReport(ReceiverReport* report);
 		void AddSdesChunk(SdesChunk* chunk);
-		void Serialize();
+		void Serialize(uint8_t* data);
 
 	private:
-		uint8_t* raw = nullptr;
-		size_t size = 0;
```

Comment:
Same as above:

* `this->raw` must exist (can be assigned once `Serialize()` is called) so `GetRaw()` returns it.
* Not sure why `size` has been removed. Also, if it represents the size in bytes, could it be renamed to `length`?